### PR TITLE
DOC-2872 ORA known limitation: can't view as specific student

### PR DESCRIPTION
**Pull request by catong**
## [DOC-2872](https://openedx.atlassian.net/browse/DOC-2872)

This PR updates the "View as Specific Student" topic in the B&R guide with a note indicating that ORA problems are not included in the types of problems that can be viewed in this way. (The case is also handled in the UI with a warning "This type of component cannot be shown while viewing the course as a specific student.)
### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 
- [ ] Requester: @jaakana
- [x] Subject matter expert: @cahrens 
- [x] Doc team review (sanity check/copy edit/dev edit): @lamagnifica @srpearce @pdesjardins 
- [x] Product review: @sstack22 
### Testing
- [ ] Ran ./run_tests.sh without warnings or errors
### Post-review
- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [x] Squash commits

### CODE REVIEW

**Comment by sstack22** [comment]
Looks good! 👍

File: en_us/shared/developing_course/testing_courseware.rst [hunk 1]
```diff
@@ -178,8 +178,8 @@ answer other than the correct one gets a red X for incorrect.
 Student View
 ============
 
-In the LMS, to view your live course as learners see it, from **View this
-course as** select **Student**.
+In the LMS, to view your live course as learners see it, select **Student** from
+the **View this course as** list.
```
Comment (lamagnifica) [inline en_us/shared/developing_course/testing_courseware.rst:183]: I prefer the older sequencing, since it presents what you have to do in the order you have to do it (find the dropdown, then choose an option from it)

Comment (lamagnifica) [inline en_us/shared/developing_course/testing_courseware.rst:183]: if you prefer this new sequencing, please change on lines 227-8 as well to keep consistent

File: en_us/shared/developing_course/testing_courseware.rst [hunk 2]
```diff
@@ -238,6 +238,8 @@ aspects of your course.
```
Comment (lamagnifica) [inline en_us/shared/developing_course/testing_courseware.rst:237]: maybe reword line 230-1 a bit, since mostly this list is of limitations to what you can do rather than actually what "you can test"(?) 
When you view content as a specific student does, be aware of the following limitations. (?)

**Comment by lamagnifica** [comment]
a couple of suggestions, 👍

File: en_us/shared/developing_course/testing_courseware.rst (hunk 1 above)
Comment (catong) [inline en_us/shared/developing_course/testing_courseware.rst:183]: The old version was difficult/ambiguous to parse. I could play with punctuation to see if it helps.

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/1084

**Comment by cahrens** [comment]
👍
